auth/src/main/kotlin/com/hedvig/android/auth/network/ResendOtpResponse.kt

```diff
@@ -0,0 +1,13 @@
+package com.hedvig.android.auth.network
+
+import com.hedvig.android.auth.ResendOtpResult
+import okhttp3.Response
+
+fun toResendOtpResult(response: Response): ResendOtpResult {
+  val responseBody = response.body?.string()
+  return if (response.isSuccessful && responseBody != null) {
```

Comment:
The spec for this reads:
```
POST <auth-base>/member-login/{id}/otp-resend
// no input, no output
```
Does `no output` also mean that responseBody is null, or how does this get shown exactly in reality I wonder? Just so we don't interpret a successful but null body as error while we don't need to.

Reply:
Good catch, it should be enough to just check for successful response here